server/models/database.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  last_name VARCHAR(50) NOT NULL,
+  first_name VARCHAR(50) NOT NULL,
+  email VARCHAR(255) UNIQUE NOT NULL,
+  password VARCHAR(50) NOT NULL,
+  city VARCHAR(50) NOT NULL,
+  state_province VARCHAR(50) NOT NULL,
+  country VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE categories (
+  id INT PRIMARY KEY NOT NULL,
+  label VARCHAR(30) NOT NULL
+);
+
+CREATE TABLE reports (
+  id SERIAL PRIMARY KEY NOT NULL,
+  user_id INT REFERENCES users (id) NOT NULL,
```

Comment:
What if a user wants to delete their account? Will we assign some specific ID?

Reply:
I have added a 'deleted' flag to the users relation. Now, if a user deletes their account, their user record will not be deleted and their 'deleted' flag will be set to TRUE. The user field for an issue will be displayed as [deleted] (like reddit) if the user has been deleted.

Reply:
Good suggestion. Thank you!

Reply:
I'll leave this conversation unresolved and wait for your approval.

Reply:
This definitely meets the requirement but you aren't really deleting peoples information if they delete their account ahah. Another alternative is transfering the report to a special user in the database.

Reply:
Yea I guess it will depend on how AWS Cognito will handle account deletion.

A lot of people said this is a good way to do it unless you wanted to have a deleted_users table, which only seemed to be useful for purging entries after a certain amount of time.

I'll leave this for now and address it when implementing AWS Cognito.

Thanks!